Dockerfile

```diff
@@ -19,6 +19,8 @@ RUN zypper ref && \
            libxml2-devel nodejs libmysqlclient-devel postgresql-devel libxslt1 && \
     zypper -n in --no-recommends -t pattern devel_basis && \
     gem install bundler --no-ri --no-rdoc -v 1.15.4 && \
```

Comment:
Do we still need this?

Reply:
Yes, since bundler does not come from the `opensuse/ruby:2.5` image, we have to fetch it through `gem`.

Reply:
But rubygems does not install it in `/usr/bin/bundle.ruby2.5`. So it won't be the one used anyway, no?

Reply:
No, the `gem` command will install /usr/bin/bundle.ruby2.5, because the gem that comes from `opensuse/ruby:2.5` right now is honoring the `.ruby2.5` suffix. So, weirdly enough:

```
$ gem install bundler --no-ri --no-rdoc -v 1.15.4
```

Will install `/usr/bin/bundle.ruby2.5` 😅

Reply:
In order to check this, I modified the Dockerfile so the last command was: `zypper -n in --no-recommends -t pattern devel_basis`, then I exec'd there and manually checked that there was no bundler nor bundler.ruby2.5, and then after using `gem` the one with the suffix existed, so a call to `update-alternatives` was needed.

Reply:
Really? Wat... Okay then.